jax/_src/pallas/pallas_call.py

```diff
@@ -77,16 +77,15 @@
 
 def _pallas_call_impl(*args, **params):
   # Call the lowering path
-  if config.disable_jit.value:
-    raise NotImplementedError(
-        "pallas_call not supported with disable_jit. Consider invoking under a"
-        " local context of `jax.disable_jit(False)`."
-    )
-
   @partial(jax.jit, inline=True)
   def _jit_run(*args):
     return pallas_call_p.bind(*args, **params)
-  return _jit_run(*args)
+
+  if config.disable_jit.value:
```

Comment:
nit: Do you need to branch on the disable_jit value? What happens if you just replace lines 84-88 with:
```
with config.disable_jit(False):
  return _jit_run(*args)
```